common/build.gradle

```diff
@@ -109,7 +109,7 @@ dependencies {
         transitive = false
     }
 
-    distApi("com.microsoft.identity:common4j:1.0.0-RC5") {
+    distApi("com.microsoft.identity:common4j:0.0.+") {
```

Comment:
So this would no longer be something like 0.0.n and rather be `0.0.$(Date:yyyyMMdd)$(Rev:.r) # $(Build.BuildNumber)` - have we verified that Android Studio can automatically pick up new ones correctly? Just want to make sure we have it verified

Reply:
Yes,